recipes-browser/cog/cog_0.10.0.bb

```diff
@@ -0,0 +1,11 @@
+require cog.inc
+require conf/include/devupstream.inc
+
+SRC_URI = "https://github.com/Igalia/cog/releases/download/v${PV}/cog-${PV}.tar.xz"
+
+SRC_URI[sha256sum] = "2c72369c636ca4684370adad1344071b23c9ee2c851eb7d738fa2e1d7092031f"
+
+SRC_URI_class-devupstream = "git://github.com/Igalia/cog.git;protocol=https;branch=master"
```

Comment:
The `_class-devupstream` can not be moved to the `.inc` file because this is share in by different versions but the `class-devupstream` can be only in one recipe.

Reply:
Ah, you are right. Bitbake is weird, I don't know why I still expect it to be logical 🤷🏼